plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/glue/GlueHiveMetastore.java

```diff
@@ -254,10 +254,10 @@ private static AWSCredentialsProvider getAwsCredentialsProvider(GlueHiveMetastor
         return provider;
     }
 
-    private static AWSCredentialsProvider getCustomAWSCredentialsProvider(String providerClass)
+    private static AWSCredentialsProvider getCustomAWSCredentialsProvider(String providerClass, String awsCredentialsProviderConf)
     {
         try {
-            Object instance = Class.forName(providerClass).getConstructor().newInstance();
```

Comment:
This is backward incompatible change. I think in case no `conf` is given we should call previous constructor.

Reply:
Updated the PR.